engine/admin-api/adapter/service/version.go

```diff
@@ -161,11 +171,24 @@ func versionToWorkflows(version *entity.Version) []*versionpb.Workflow {
 			Name:       w.Name,
 			Entrypoint: w.Entrypoint,
 			Nodes:      nodes,
-			Edges:      edges,
+			Exitpoint:  w.Exitpoint,
+			Stream:     workflowStreamConfig.Stream,
+		}
+
```

Comment:
flag as to deprecate
```suggestion

// TODO krt-v1: deprecate retrocompatibility
```